setup.py

```diff
@@ -53,14 +53,23 @@ def read(filename):
     'dev': [
         'wheel>=0.29.0',
         'twine>=1.8.1',
-        'pytest==3.0.4',
-        'pytest-cov==2.4.0',
-        'pytest-flake8==0.8.1',
-        'pytest-mock==1.4.0',
-        'flake8==3.3.0'
-    ]
+    ],
 }
 
+tests_require = [
+    'pytest==3.0.4',
+    'pytest-cov==2.4.0',
+    'pytest-flake8==0.8.1',
+    'pytest-mock==1.4.0',
+    'flake8==3.3.0',
+]
+
```

Comment:
I think it'd be good to eliminate this space to make it clear from a glance that this code is grouped with the `tests_require` above.

Reply:
Removed.